packages/block-library/src/index.js

```diff
@@ -45,7 +45,7 @@ import * as pullquote from './pullquote';
 import * as reusableBlock from './block';
 import * as rss from './rss';
 import * as search from './search';
-import * as section from './section';
+import * as group from './group';
```

Comment:
Can it be moved to be sorted alphabetically?

Reply:
👍 done